build/emoji/emoji_gen.mjs

```diff
@@ -285,7 +285,15 @@ fullEmoji.forEach((emoji, index) => {
     emoji.image = file;
 
     if (emoji.category !== 'custom') {
-        emojiFilePositions.set(file, `-${emoji.sheet_x * EMOJI_SIZE_PADDED}px -${emoji.sheet_y * EMOJI_SIZE_PADDED}px;`);
```

Comment:
The CSS generated by this script was last updated before we added a style checker for CSS, so that file had been manually made to fit the style checker. I updated this to ensure the generated CSS was valid